scripts/resultsSum.pl

```diff
@@ -241,8 +242,14 @@ sub resultReporter {
 		print "<Jenkins URL>/parambuild/?JDK_VERSION=$jdkVersion&JDK_IMPL=$jdkImpl$buildParam$platformParam$customTargetParam&TARGET=<FAILED test target>\n\n";
 		print "For example, to rebuild the failed tests in <Jenkins URL>=${hudsonUrl}job/Grinder, use the following links:\n";
 		foreach my $failedTarget (@failed) {
+			$failedList = $failedList . ${failedTarget} . ",";
 			print "${hudsonUrl}job/Grinder/parambuild/?JDK_VERSION=$jdkVersion&JDK_IMPL=$jdkImpl$buildParam$platformParam$customTargetParam&TARGET=$failedTarget\n";
 		}
+		if ($numOfFailed > 1) {
+			chop($failedList);
```

Comment:
Maybe it's more concise to use `join(",", @failed)` than creating a `$faliledList`.